kolibri/utils/server.py

```diff
@@ -305,6 +305,9 @@ def SERVING(self, port):
         else:
             self.broadcast.update_broadcast(instance=instance)
 
+        # cache the list of addresses active right now, so we can detect changes
+        self.addresses = get_all_addresses()
```

Comment:
This is already cached within Zeroconf so why cache again? Also, this breaks the class API of `KolibriBroadcast` since this is outside of what the broadcast was initialized with.